test-framework/test-suites/integration/tests/report/test_report_host_bootfile.py

```diff
@@ -0,0 +1,145 @@
+import pytest
+
+@pytest.mark.usefixtures('revert_database')
+class TestReportHostBootfile:
+	def test_report_host_bootfile_on_fresh_install(self, host):
+		result = host.run('stack report host bootfile')
+		assert result.rc == 0
+
+	def test_report_host_bootfile_for_host_with_full_IP_config(self, host):
```

Comment:
Can this use the add_host_interface test fixture?  This would remove 5 lines of code from this test whose effects you aren't actually testing.